tools/generatechangelog/generatechangelog_test.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/AlekSi/pointer"
+	"github.com/FerretDB/gh"
+	"github.com/google/go-github/v57/github"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetMilestone(t *testing.T) {
+	ctx := context.Background()
+	client, err := gh.NewRESTClient(os.Getenv("GITHUB_TOKEN"), nil)
```

Comment:
As the tests (this one and the one below) work with real GitHub API, ideally, we need to set `GITHUB_TOKEN` on CI when running tests. Otherwise, we might be rate-limited.

What do you think?